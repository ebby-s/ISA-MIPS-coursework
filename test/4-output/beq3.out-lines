TB : INFO : register_v0=00000065
TB : finished; active=0
