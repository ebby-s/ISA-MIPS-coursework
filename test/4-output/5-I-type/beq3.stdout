Loading test/2-binary/5-I-type/beq3.hex.txt into RAM
VCD info: dumpfile mips_cpu_bus_tb.vcd opened for output.
TB : INFO : RAM_ACCESS: Read from 0xbfc00000, data: 0x24630001
TB : INFO : RAM_ACCESS: Read from 0xbfc00004, data: 0x26100014
TB : INFO : RAM_ACCESS: Read from 0xbfc00008, data: 0x26520014
TB : INFO : RAM_ACCESS: Read from 0xbfc0000c, data: 0x1212fffd
TB : INFO : RAM_ACCESS: Read from 0xbfc00010, data: 0x2652000a
TB : INFO : RAM_ACCESS: Read from 0xbfc00004, data: 0x26100014
TB : INFO : RAM_ACCESS: Read from 0xbfc00008, data: 0x26520014
TB : INFO : RAM_ACCESS: Read from 0xbfc0000c, data: 0x1212fffd
TB : INFO : RAM_ACCESS: Read from 0xbfc00010, data: 0x2652000a
TB : INFO : RAM_ACCESS: Read from 0xbfc00014, data: 0x02121021
TB : INFO : RAM_ACCESS: Read from 0xbfc00018, data: 0x00431021
TB : INFO : RAM_ACCESS: Read from 0xbfc0001c, data: 0x0000f808
TB : INFO : RAM_ACCESS: Read from 0xbfc00020, data: 0x00000000
TB : INFO : RAM_ACCESS: Read from 0x00000000, data: 0x00000000
TB : INFO : register_v0=00000065
TB : finished; active=0
